### PR TITLE
Add see it in action sections to sample readme

### PR DESCRIPTION
**Pull request by dbjorge**
#### Description of changes

Adds "see it in action" instructions and screenshots to give the readme a little more pizzaz.

Also removed obsolete http-server reference since I noticed it was still around while moving readme elements around (we replaced it in an earlier PR with just pointing selenium to a file:// url directly)

#### Pull request checklist

- [x] If this PR addresses an existing issue, it is linked: Fixes #0000
- [x] `yarn test` passes in all affected samples
- [x] Added any applicable tests

### CODE REVIEW

File: typescript-selenium-webdriver/README.md [hunk 1]
```diff
@@ -1,17 +1,57 @@
...
-* [typescript](https://www.typescriptlang.org/) and [jest](https://jestjs.io/) to write tests of the page
+* [typescript](https://www.typescriptlang.org/) to author our test code
+* [Jest](https://jestjs.io/) as our test framework, with [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing) for baselining
 * [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver) to automate browsing to the page from the tests
 * [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs) to run an axe accessibility scan on the page from the Selenium browser
 * [axe-sarif-converter](https://github.com/microsoft/axe-sarif-converter) and [Sarif.Multitool](https://www.nuget.org/packages/Sarif.Multitool) to compare the scan results to a checked-in baseline
 * [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to run the tests in a CI build with every Pull Request
 * [Sarif Viewer Build Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.sarif-viewer-build-tab) to visualize the baselined results in Azure Pipelines
+
+## See it in action In Azure Pipelines
```
Comment (lisli1) [inline typescript-selenium-webdriver/README.md]: Minor nit: "In" to "in" (capital to lowercase)

File: typescript-selenium-webdriver/README.md [hunk 2]
```diff
@@ -1,17 +1,57 @@
...
+
+   ```shell
+   git clone https://github.com/microsoft/axe-pipelines-samples
+   ```
+
+1. Install the dependencies
+
+   ```shell
+   cd ./typescript-selenium-webdriver
+   npm install
```
Comment (lisli1) [inline typescript-selenium-webdriver/README.md]: I thought we were using yarn instead of npm in this sample

Comment (dbjorge) [reply typescript-selenium-webdriver/README.md]: oops, good catch. Will update to clarify that both are fine

Comment (smoralesd) [reply typescript-selenium-webdriver/README.md]: does `yarn` needs to show up on the Tools and library section? or is that something we consider given or obvious?

Comment (peterdur) [reply typescript-selenium-webdriver/README.md]: I think we want to err on the side of stating the obvious for samples like this. `jest` also needs to be globally installed for the local machine instructions to work (or you can `yarn run jest`).

File: typescript-selenium-webdriver/README.md [hunk 3]
```diff
@@ -1,17 +1,57 @@
...
+1. Clone this sample repository
+
+   ```shell
+   git clone https://github.com/microsoft/axe-pipelines-samples
+   ```
+
+1. Install the dependencies
+
+   ```shell
+   cd ./typescript-selenium-webdriver
```
Comment (peterdur) [inline typescript-selenium-webdriver/README.md]: Incorrect directory name?

**Pull request merged**